src/digest.clj

```diff
@@ -1,10 +1,8 @@
 (ns digest
-  #^{ :author "Miki Tebeka <miki.tebeka@gmail.com>"
```

Comment:
Why remove the meta data (OK for my name, but the doc?)

Reply:
Oh, I'm sorry about removing your name and doc from require - i executed `lein slamhound` in project folder which optimised the require but also removed the meta data. Will restore it!

Reply:
`(:require .. :refer ..)` is a new way to do the same thing that allows effectively deprecate `:use`, which has some drawbacks. Please see https://stackoverflow.com/questions/871997/difference-between-use-and-require#comment25370819_16429572 for more details.